<launch>
  <node pkg="smallBotRosDriver"
   name="driver_node" type="driver_node" output="screen">
    <param name="port" type="str" value="/dev/ttyAMA0" />
    <param name="cmd" type="str" value="/cmd_vel"/>
    <param name="baud_rate" type="int" value="115200"/>
    <param name="timeout_millseconds" type="int" value="2"/>
    <param name="receive_queue_size"  type="int" value="100"/>
    <param name="reductionRate"  type="double" value="0.033"/>
    <param name="L"  type="double" value="0.015"/>
    <param name="wheelR"  type="double" value="0.029"/>
  </node>
</launch>